Detectors/TPC/calibration/include/TPCCalibration/CalibdEdxBase.h

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file CalibdEdxBase.h
+/// \brief
+/// \author Thiago Badaró <thiago.saramela@usp.br>
+
+#ifndef ALICEO2_TPC_CALIBDEDXBASE_H_
+#define ALICEO2_TPC_CALIBDEDXBASE_H_
+
+#include <array>
+#include <cstddef>
+
+namespace o2::tpc
+{
+
+/// Define the commom properties of dEdx containers. Used by CalibdEdxHistos and CalibdEdx.
+template <typename Entry>
+class CalibdEdxBase
+{
+ public:
+  static const size_t stacksPerTurn = 18;
+  static const size_t totalStacks = 18 * 8;
```

Comment:
Please check information in 
https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Detectors/TPC/include/DataFormatsTPC/Defs.h
```suggestion
static const size_t totalStacks = SECTORSPERSIDE * SIDES * GEMSTACKSPERSECTOR;```
```
your `stacksPerTurn` is `SECTORSPERSIDE`